starting worker pid=4540 on localhost:11644 at 08:51:57.865
starting worker pid=8892 on localhost:11644 at 08:51:57.963
starting worker pid=380 on localhost:11644 at 08:51:58.054
starting worker pid=2612 on localhost:11644 at 08:51:58.145
starting worker pid=14080 on localhost:11644 at 08:51:58.235
starting worker pid=10884 on localhost:11644 at 08:51:58.324
starting worker pid=5504 on localhost:11644 at 08:51:58.568
starting worker pid=11080 on localhost:11644 at 08:51:58.663
starting worker pid=11168 on localhost:11644 at 08:51:58.757
starting worker pid=8660 on localhost:11644 at 08:51:58.850
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=6064 on localhost:11616 at 18:33:32.876
starting worker pid=11904 on localhost:11616 at 18:33:32.975
starting worker pid=11116 on localhost:11616 at 18:33:33.066
starting worker pid=15940 on localhost:11616 at 18:33:33.160
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=13092 on localhost:11976 at 19:46:02.318
starting worker pid=9332 on localhost:11976 at 19:46:02.414
starting worker pid=16076 on localhost:11976 at 19:46:02.503
starting worker pid=2612 on localhost:11976 at 19:46:02.592
starting worker pid=1092 on localhost:11976 at 19:46:02.681
starting worker pid=8168 on localhost:11976 at 19:46:02.769
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialiExecution halted
or reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=12932 on localhost:11385 at 20:45:57.366
starting worker pid=13112 on localhost:11385 at 20:45:57.463
starting worker pid=2300 on localhost:11385 at 20:45:57.551
starting worker pid=11520 on localhost:11385 at 20:45:57.638
starting worker pid=9080 on localhost:11385 at 20:45:57.726
starting worker pid=4724 on localhost:11385 at 20:45:57.818
starting worker pid=13124 on localhost:11385 at 20:45:57.906
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
starting worker pid=6268 on localhost:11281 at 20:48:01.693
starting worker pid=9036 on localhost:11281 at 20:48:01.851
starting worker pid=11624 on localhost:11281 at 20:48:01.995
starting worker pid=9276 on localhost:11281 at 20:48:02.129
starting worker pid=1472 on localhost:11281 at 20:48:02.265
starting worker pid=5828 on localhost:11281 at 20:48:02.427
starting worker pid=12940 on localhost:11281 at 20:48:02.582
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=13712 on localhost:11192 at 21:39:39.857
starting worker pid=10324 on localhost:11192 at 21:39:39.947
starting worker pid=9664 on localhost:11192 at 21:39:40.038
starting worker pid=7620 on localhost:11192 at 21:39:40.129
starting worker pid=1332 on localhost:11192 at 21:39:40.218
starting worker pid=2960 on localhost:11192 at 21:39:40.308
starting worker pid=11412 on localhost:11192 at 21:39:40.397
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=14364 on localhost:11360 at 22:06:16.660
starting worker pid=13656 on localhost:11360 at 22:06:16.755
starting worker pid=14372 on localhost:11360 at 22:06:16.845
starting worker pid=11760 on localhost:11360 at 22:06:16.941
starting worker pid=13932 on localhost:11360 at 22:06:17.033
starting worker pid=17248 on localhost:11360 at 22:06:17.122
starting worker pid=13928 on localhost:11360 at 22:06:17.221
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialiExecution halted
or reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=15348 on localhost:11947 at 07:15:07.731
starting worker pid=3192 on localhost:11947 at 07:15:07.830
starting worker pid=13624 on localhost:11947 at 07:15:07.920
starting worker pid=8800 on localhost:11947 at 07:15:08.008
starting worker pid=4556 on localhost:11947 at 07:15:08.095
starting worker pid=9568 on localhost:11947 at 07:15:08.183
starting worker pid=15412 on localhost:11947 at 07:15:08.272
starting worker pid=10072 on localhost:11947 at 07:15:08.361
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=12712 on localhost:11996 at 10:32:33.806
starting worker pid=12696 on localhost:11996 at 10:32:33.901
starting worker pid=14924 on localhost:11996 at 10:32:33.990
starting worker pid=5980 on localhost:11996 at 10:32:34.080
starting worker pid=15608 on localhost:11996 at 10:32:34.175
starting worker pid=6704 on localhost:11996 at 10:32:34.269
starting worker pid=3176 on localhost:11996 at 10:32:34.361
starting worker pid=9612 on localhost:11996 at 10:32:34.450
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
ze(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=7024 on localhost:11996 at 11:42:04.616
starting worker pid=3160 on localhost:11996 at 11:42:04.712
starting worker pid=14376 on localhost:11996 at 11:42:04.808
starting worker pid=15592 on localhost:11996 at 11:42:04.904
starting worker pid=4468 on localhost:11996 at 11:42:04.999
starting worker pid=14952 on localhost:11996 at 11:42:05.093
starting worker pid=5868 on localhost:11996 at 11:42:05.188
starting worker pid=2868 on localhost:11996 at 11:42:05.284
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=4048 on localhost:11996 at 11:42:29.423
starting worker pid=13684 on localhost:11996 at 11:42:29.517
starting worker pid=15016 on localhost:11996 at 11:42:29.610
starting worker pid=5952 on localhost:11996 at 11:42:29.699
starting worker pid=9480 on localhost:11996 at 11:42:29.789
starting worker pid=5488 on localhost:11996 at 11:42:29.880
starting worker pid=10112 on localhost:11996 at 11:42:29.969
starting worker pid=15600 on localhost:11996 at 11:42:30.051
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
ze(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=6276 on localhost:11996 at 11:43:08.772
starting worker pid=8864 on localhost:11996 at 11:43:08.864
starting worker pid=11260 on localhost:11996 at 11:43:08.953
starting worker pid=2596 on localhost:11996 at 11:43:09.041
starting worker pid=15260 on localhost:11996 at 11:43:09.131
starting worker pid=3272 on localhost:11996 at 11:43:09.217
starting worker pid=2036 on localhost:11996 at 11:43:09.311
starting worker pid=15412 on localhost:11996 at 11:43:09.401
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=11444 on localhost:11996 at 11:43:21.424
starting worker pid=2420 on localhost:11996 at 11:43:21.521
starting worker pid=6200 on localhost:11996 at 11:43:21.606
starting worker pid=7776 on localhost:11996 at 11:43:21.702
starting worker pid=6780 on localhost:11996 at 11:43:21.793
starting worker pid=5228 on localhost:11996 at 11:43:21.882
starting worker pid=11748 on localhost:11996 at 11:43:21.973
starting worker pid=13500 on localhost:11996 at 11:43:22.063
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
